src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -522,3 +542,59 @@ export async function validate8BitsTiff(tiff: Tiff): Promise<void> {
     throw new Error(`${tiff.source.url.href} is not a 8 bits TIFF`);
   }
 }
+
+/**
+ * Get the list of map sheets / tiles that intersect with the given bounding box.
+ *
+ * @param bbox Bounding box of the area of interest (in EPSG:2193) to get the map sheets for (e.g. TIFF area).
+ * @param gridSize Grid size of the map sheets / tiles to get.
+ * @param minIntersectionMeters Minimum intersection area in meters (width or height) to include the map sheet.
+ */
+function getCovering(bbox: BBox, gridSize: GridSize, minIntersectionMeters = 0.15): string[] {
+  const SurroundingTiles = [
+    { x: 1, y: 0 },
+    { x: 0, y: -1 }, // inverted Y axis
+    // Future versions may want to explore tiles in all directions
```

Comment:
might be safer to leave these in? is there any downside? I feel like we could check a few million intersections a second?

Reply:
done